javascript/src/stores/search/SearchStore.ts

```diff
@@ -238,6 +238,13 @@ class SearchStore {
         searchURLParams = searchURLParams.set(param, value);
         URLUtils.openLink(jsRoutes.controllers.SearchControllerV2.index().url + "?" + Qs.stringify(searchURLParams.toJS()));
     }
+
+    getCsvExportURL(): string {
+        var searchURLParams = this.getOriginalSearchURLParams();
+        searchURLParams = searchURLParams.delete('page');
+        // function (q,filter,rangetype,relative,from,to,keyword,fields)
```

Comment:
It was left over, I'll remove it. It's the signature for the javascript router for that controller action.